android/app/src/main/AndroidManifest.xml

```diff
@@ -1,62 +1,75 @@
-<manifest
-  xmlns:android="http://schemas.android.com/apk/res/android"
-  package="com.dismoi.scout"
->
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
It feels like these diffs are present in every PR but I might be wrong :sweat_smile: